docs/tutorials/podman_tutorial_cn.md

```diff
@@ -0,0 +1,238 @@
+> - 译文出自：[掘金翻译计划](https://juejin.cn/translate)
+
+
+
+![PODMAN logo](https://github.com/containers/podman/raw/main/logo/podman-logo-source.svg)
+
+Podman是作为libpod库的一部分提供的程序，可以被用于创建和管理容器。
+
+下面的教程会教你如何启动 Podman 并使用 Podman 执行一些基本的命令。
+
+
+如果你正在使用 Mac 或者 Windows ，你应该先查看[Mac和Windows使用说明](https://github.com/containers/podman/blob/master/docs/tutorials/mac_win_client.md)来设置 Podman远程客户端。
+
+
+
+**注意**：这些代码示例旨在作为非root的用户运行，并且在需要root权限的时候使用`sudo`。
+
+
+## 安装Podman
+
+
+
+安装或者构建Podman，请参照[安装说明](https://github.com/containers/podman/blob/master/install.md)。
+
+
+
+## 熟悉podman
+
+
+### 运行一个示例容器
+
+
+这个示例容器会运行一个简单的只有主页的httpd服务器。
+
+```bash
+podman run -dt -p 8080:8080/tcp -e HTTPD_VAR_RUN=/run/httpd -e HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
+                  -e HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
+                  -e HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
+                  registry.fedoraproject.org/f29/httpd /usr/bin/run-httpd
+```
+
+
+
+
+因为命令中的`-d`参数表明容器以`detached`模式运行，所以Podman会在容器运行后打印容器的ID。
+
+注意为了访问这个HTTP服务器，我们将使用端口转发。成功运行需要slirp4nextns 的 v0.3.0+版本。
+
+
+Podman的`ps`命令用于列出正在创建和运行的容器。
+
+```console
+podman ps
+```
+
+
+
+
```

Comment:
Seems too many blank lines here. This is also happens in other places. Can you help to clean up it?

Reply:
Hello @of2x Still have 5 blank lines here. Can you help to check it?